app/templates/footer_section.html

```diff
@@ -1,17 +1,101 @@
+				<h5 class="text-uppercase mb-4 font-weight-bold text-warning">GetSetQuiz</h5>
+				<p>Passionate about Quizzes? You can engage with thousands of players 
+                from home or at different venues here! Join our discord today and enter the community of quiz enthusiasts!</p>
+				
+			</div>
+
+			<div class="col-md-2 col-lg-2 col-xl-2 mx-auto mt-3">
+				<h5 class="text-uppercase mb-4 font-weight-bold text-warning">Quiz</h5>
+			<p>
+				<a href="#" class="text-white" style="text-decoration: none;"> Host a Quiz</a>
+			</p>
+			<p>
+				<a href="#" class="text-white" style="text-decoration: none;"> Create a Quiz</a>
+			</p>
+			<p>
+				<a href="#" class="text-white" style="text-decoration: none;"> Join a Quiz</a>
+			</p>
+			
+			</div>
+
+			<div class="col-md-3 col-lg-2 col-xl-2 mx-auto mt-3">
+				<h5 class="text-uppercase mb-4 font-weight-bold text-warning">Useful links</h5>
+			<p>
+				<a href="#" class="text-white" style="text-decoration: none;"> Your Account</a>
+			</p>
+			<p>
+				<a href="#" class="text-white" style="text-decoration: none;"> Become an Affiliate </a>
+			</p>
+			<p>
+				<a href="#" class="text-white" style="text-decoration: none;"> Join Our Community</a>
+			</p>
+			
+			</div>
+
+			<div class="col-md-4 col-lg-3 col-xl-3 mx-auto mt-3">
+				<h5 class="text-uppercase mb-4 font-weight-bold text-warning" >Contact</h5>
+				<p>
+					<i class="fas fa-home mr-3"></i> IN
+				</p>
+				<p>
+					<i class="fas fa-envelope mr-3"></i>getsetquizindia@gmail.com
+				</p>
+				<p>
+					<i class="fas fa-phone mr-3"></i>+99999999
+				</p>
+				<p>
+					<i class="fas fa-print	 mr-3"></i>+99999999
+			</div>
+			
+		</div>
+
+		<hr class="mb-4">
+
+		<div class="row align-items-center">
+
+			<div class="col-md-7 col-lg-8">
+				<p>	Copyright ©2020 All rights reserved by:
+					<a href="#" style="text-decoration: none;">
+						<strong class="text-warning">GetSetQuiz</strong>
+					</a></p>
+				
+			</div>
+
+			<div class="col-md-5 col-lg-4">
+				<div class="text-center text-md-right">
+
+					<ul class="list-unstyled list-inline">
+						<li class="list-inline-item">
+							<a href="#" class="btn-floating btn-sm text-white" style="font-size: 23px;"><i class="fab fa-facebook"></i></a>
+						</li>
+						<li class="list-inline-item">
+							<a href="#" class="btn-floating btn-sm text-white" style="font-size: 23px;"><i class="fab fa-twitter"></i></a>
+						</li>
+						<li class="list-inline-item">
+							<a href="#" class="btn-floating btn-sm text-white" style="font-size: 23px;"><i class="fab fa-discord"></i></a>
+						</li>
+						<li class="list-inline-item">
+							<a href="#" class="btn-floating btn-sm text-white" style="font-size: 23px;"><i class="fab fa-linkedin-in"></i></a>
+						</li>
+						<li class="list-inline-item">
+							<a href="#" class="btn-floating btn-sm text-white" style="font-size: 23px;"><i class="fab fa-youtube"></i></a>
+						</li>
```

Comment:
if all these elements have `style=font-size: 23px;` then why cant we have css class for them?